lib/api/global.js

```diff
@@ -19,42 +20,10 @@ export async function getGlobalData() {
     ${contactFormFragment}
     ${categoriesFragment}
     ${userProfileFragment}
-    {
+    query getGlobalData($site: [String]) {
```

Comment:
previously all global data was being fetched twice and stored like:

```
data {
    global: {...},
    global_es: {...}`
}
```

This change adds the site as a param in the function and only retrieves globals for the current site, effectively halving the query size.